### DISCUSSION TITLE
Education

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#37642] [GitHub Education] [answered]
Does GitHub provide education discounts and free subscriptions on specific products?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @iLakshya. Yes, some subscriptions are provided by the Student Developer Pack, for example:

![image](https://user-images.githubusercontent.com/47532703/198663359-11750138-160f-4835-b5d7-ae9849a3142e.png)

**Reply by SilasRodrigues19** [reply]
You can see a complete list of what is offered at [https://education.github.com/pack/offers](https://education.github.com/pack/offers)

**Comment by gururajkoni04** [comment]
I recently discovered that my GitHub Education plan was revoked without any prior notification or reason. When I attempt to re-enroll via the [GitHub Education application page](https://education.github.com/discount_requests/application), I am being 301 redirected to the [GitHub pricing page](https://github.com/pricing), preventing me from submitting a new request.

Even when I try accessing the exact URL provided in the email regarding my GitHub Education benefits, it redirects in the same way, making it impossible to reapply.

I am a valid student enrolled in a recognized educational institution, which was successfully verified just a few weeks ago. Since my eligibility remains unchanged, I would appreciate your assistance in reinstating my GitHub Education benefits or providing a resolution to bypass this redirection issue.

Please let me know if any additional details or documents are required to resolve this matter.

Looking forward to your prompt response.
